.github/workflows/tests.yml

```diff
@@ -11,18 +11,16 @@ name: "Tests"
 
 jobs:
   phpunit:
-    name: "PHPUnit on ${{ matrix.operating-system }} with PHP ${{ matrix.php-version }}"
```

Comment:
Removing the reference to the PHP version here as the way it shows up when not referencing it here is totally fine.